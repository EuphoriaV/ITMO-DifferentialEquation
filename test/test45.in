1.57
(((4.72 + x) - ((((9.47 - 2.51) * (x + 4.3)) - 10.0) * (((x * 3.38) * 6.87) + 5.67))) * (x + (7.59 - x)))
5.69
1.58