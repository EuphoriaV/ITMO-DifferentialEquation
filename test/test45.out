-6.2590638